src/GuppyDataExplorer/ExplorerVisualization/index.jsx

```diff
@@ -216,6 +216,7 @@ class ExplorerVisualization extends React.Component {
                   ...heatMapGuppyConfig,
                 }}
                 filterConfig={heatMapFilterConfig}
+                hideEmptyFilterSection={hideEmptyFilterSection}
```

Comment:
you don't really need this, this filter is for DAT and it is not visible anyways

Reply:
thanks ill remove that one